### PR TITLE
[refine](pipeline) remove no pipeline code in SenderQueue

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

File: be/src/vec/runtime/vdata_stream_recvr.cpp [hunk 1]
```diff
@@ -63,18 +63,7 @@ VDataStreamRecvr::SenderQueue::~SenderQueue() {
...
-                 << " node=" << _recvr->dest_node_id();
-        // Don't count time spent waiting on the sender as active time.
-        CANCEL_SAFE_SCOPED_TIMER(_recvr->_data_arrival_timer, &_is_cancelled);
-        CANCEL_SAFE_SCOPED_TIMER(
-                _received_first_batch ? nullptr : _recvr->_first_batch_wait_total_timer,
-                &_is_cancelled);
-        _data_arrival_cv.wait(l);
-    }
+    std::lock_guard<std::mutex> l(_lock);
     return _inner_get_batch_without_lock(block, eos);
```
Comment (Gabriel39) [inline be/src/vec/runtime/vdata_stream_recvr.cpp:67]: `_inner_get_batch_without_lock` is not needed after this PR.

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run cloud_p0

**Comment by Mryange** [comment]
run buildall

**Pull request closed without merging**